app/models/concerns/taggable.rb

```diff
@@ -77,6 +77,34 @@ def freeform_string=(tag_string)
     parse_tags(Freeform, tag_string)
   end
 
+  def tag_string
+    tags.map{|tag| tag.name}.join(ArchiveConfig.DELIMITER_FOR_OUTPUT)
+  end
+
+  def tag_string=(tag_string)
```

Comment:
Can you add a comment for this method?

```
# Process a string of tags from any tag class. Unlike #parse_tags, it allows the creation of UnsortedTag instances.
# Used in bookmarks and collections.
```

(@sarken, from the changes we plan for tag wrangling pages I assume that collection owners can make unsorted tags, like bookmarkers and unlike work creators.)

Reply:
Right, any new tags created by the collection owners should be unsorted tags.